config/config/sc-config.yaml

```
@@ -1073,6 +1073,11 @@ networkPolicies:
       ips: # ip(s) to wc loadbalancer if available, otherwise wc worker nodes
         - set-me
     externalLoadBalancer: false # only true if loadbalancer is not controlled by a kubernetes cloud controller
+    objectStorage:
+      ips: # ips to s3 service
+      - set-me
```

```suggestion
      # ips to s3 service
      ips:
      - set-me
```
@lucianvlad and I found a weird bug that if an empty sequence is merged into a node containing an inline comment then it will render invalid yaml. So for the time being please refrain from inline comments on sequence nodes.

---

Okay :+1: